modules/stochastic_tools/test/tests/surrogates/nearest_point/predictor_response.i

```diff
@@ -17,6 +17,10 @@
 []
 
 [VectorPostprocessors]
+  [sampler_data]
+    type = SamplerData
+    sampler = sample
+  []
```

Comment:
I thought your were going to add automatic handling of the sampler data by default in the SamplerTrainer, such that it has a _current_sampler_row (or something) that is automatically updated. That would allow you to just use this directly in the NearestPointTrainer and not have to add this.  NearestPointTrainer should only need "sampler_cols" that would be used to index into _current_sampler_rot.

Reply:
I did do the automatic handling. This is only to test the feature of using a reporter value as a predictor.

Reply:
Thanks. I will look over this in more detail today.